mobile-app/app/screens/AppNavigator/screens/Balances/BalancesScreen.tsx

```diff
@@ -57,57 +60,77 @@ export function BalancesScreen ({ navigation }: Props): JSX.Element {
   }, [height, wallets])
 
   useEffect(() => {
+    fetchPortfolioData()
```

Comment:
Should we also have isFocused here?

Reply:
we fetching token data on continuous basis so that we will have latest balance details on all screens, so here we are not adding is focused flag here, added note in the code